ExilenceNextApp/src/components/settings-tabs/number-input-setting/NumberInputSetting.tsx

```diff
@@ -7,29 +7,60 @@ import {
 } from '@material-ui/core';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import MinuteFormat from '../../minute-format/MinuteFormat';
 import useStyles from './NumberInputSetting.styles';
+import NumberFormat from 'react-number-format';
 
 interface Props {
   value: number;
   handleChange: (value: number) => void;
   translationKey: string;
   requiresSnapshot?: boolean;
-  minutes?: boolean;
+  suffixKey?: string,
+  minimum: number,
+  maximum: number,
   disabled?: boolean;
 }
 
+const NumberInputFormat = (props: any) => {
```

Comment:
Possible to type this? (props: Props)

Reply:
I just copied that from the old MinuteFormat, was unsure about the type.

Reply:
Ah right, I'll see if I can type it later, will resolve